gitpod-io/gitpod
# Remove project-related code

## Description

This PR does the following, all related to cleaning up after Projects:
- It removes the actual frontend code for Projects, along with all `/projects/` routes in the router
- It revamps the admin UI to use the new UI components (pending a followup for migrating the nomenclature and API methods to use the `Configuration` and `Prebuild` types from the API (EXP-1858).

## Related Issue(s)

Fixes EXP-1857

## How to test

https://ft-remove-projects.preview.gitpod-dev.com/workspaces

1. Make sure redirects are registered properly (try `/projects/`)
2. (optional) make yourself an admin and make sure the projects view continues to work

## Documentation


#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

</details>

<details>
<summary>Installer</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Saves cost. Untick this only if you're really sure you need a non-preemtible machine.
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
</details>

/hold
